provider/resources.go

```diff
@@ -104,9 +119,11 @@ func Provider() tfbridge.ProviderInfo {
 					},
 				},
 			},
-			"pagerduty_event_rule":           {Tok: makeResource(mainMod, "EventRule")},
-			"pagerduty_extension":            {Tok: makeResource(mainMod, "Extension")},
-			"pagerduty_extension_servicenow": {Tok: makeResource(mainMod, "ExtensionServiceNow")},
+			"pagerduty_event_rule":                {Tok: makeResource(mainMod, "EventRule")},
+			"pagerduty_extension":                 {Tok: makeResource(mainMod, "Extension")},
+			"pagerduty_extension_servicenow":      {Tok: makeResource(mainMod, "ExtensionServiceNow")},
+			"pagerduty_incident_workflow":         {Tok: makeResource(mainMod, "PagerdutyIncidentWorkflow")},
+			"pagerduty_incident_workflow_trigger": {Tok: makeResource(mainMod, "PagerdutyIncidentWorkflowTrigger")},
```

Comment:
Why do these resources have a "Pagerduty" prefix?

```suggestion
			"pagerduty_incident_workflow":         {Tok: makeResource(mainMod, "IncidentWorkflow")},
			"pagerduty_incident_workflow_trigger": {Tok: makeResource(mainMod, "IncidentWorkflowTrigger")},
```